CHANGELOG.md

```diff
@@ -3,6 +3,89 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.20.0
+
+_Oct 10, 2024_
+
+We'd like to offer a big thanks to the 14 contributors who made this release possible. Here are some highlights ✨:
+
+- 📚 Updated ["What's new"](https://mui.com/x/whats-new/) page giving more detailed overview of the latest new features and other highlights
+- 📚 New [collapsible column groups demo](https://mui.com/x/react-data-grid/column-groups/#collapsible-column-groups) for the Data Grid component
+- 🌍 Improved Japanese (ja-JP) locale on the Data Grid component
+- 🐞 Bugfixes
+- 📚 Other documentation improvements
+
+Special thanks go out to our community contributors who have helped make this release possible:
+@k-rajat19, @kalyan90, @uma-neko, @vfbiby.
+Following are all team members who have contributed to this release: @alelthomas, @arminmeh,
+@cherniavskii, @flaviendelangle, @JCQuintas, @MBilalShafi, @noraleonte, @oliviertassinari, @samuelsycamore, @siriwatknp.
+
+<!--/ HIGHLIGHT_ABOVE_SEPARATOR /-->
+
+### Data Grid
+
+#### `@mui/x-data-grid@7.20.0`
+
+- [DataGrid] Add `onColumnHeaderContextMenu` event (#14734) @vfbiby
+- [DataGrid] Avoid row spanning computation of outdated rows (#14902) @MBilalShafi
+- [DataGrid] Fix scrollbar position not being updated after `scrollToIndexes` (#14888) @arminmeh
+- [DataGrid] Pass `rowId` param to `processRowUpdate` (#14821) @k-rajat19
+- [l10n] Improve Japanese (ja-JP) locale (#14870) @uma-neko
+
+#### `@mui/x-data-grid-pro@7.20.0` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-data-grid@7.20.0`.
+
+#### `@mui/x-data-grid-premium@7.20.0` [![premium](https://mui.com/r/x-premium-svg)](https://mui.com/r/x-premium-svg-link 'Premium plan')
+
+Same changes as in `@mui/x-data-grid-pro@7.20.0`.
+
+### Date and Time Pickers
+
+#### `@mui/x-date-pickers@7.20.0`
+
+- [pickers] Fix regression on `PickerValidDate` (#14896) @flaviendelangle
+- [pickers] Move the `DateFieldInPickerProps` interface to the `DatePicker` folder and rename it `DatePickerFieldProps` (same for time and date time) (#14828) @flaviendelangle
+
+#### `@mui/x-date-pickers-pro@7.20.0` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-date-pickers@7.20.0`.
+
+### Charts
+
+#### `@mui/x-charts@7.20.0`
```

Comment:
Anything to wait for here?

Reply:
nope

Reply:
should I remove the charts completely from the changelog or add
`No updates since @mui/x-charts@7.19.0`.

Reply:
![image](https://github.com/user-attachments/assets/9bd5425d-d26b-4ed9-83b1-20551d39fd24)


We kept it for a previous empty of the Tree View, but it's super ugly
If you can apply the same logic as what we chose for the charts on today's release it would be great

I would be in favor of removing it entirely if we don't release it and keeping it with "No updates" if we don't release it.
Having it in the list if we don't release it is weird.
And if we release I think it's nice to have it but no strong opinion here.

Reply:
I agree that the changelog appearance and the release should be coupled

[Semver does not require incrementing versions by 1](https://github.com/semver/semver/issues/356#issuecomment-347736182), so we can skip releasing the package and not include it in the changelog. Next release, the version can be aligned with the other packages.

Reply:
@arminmeh Skipping releasing packages backfired a few times already, I strongly advise not to do so 😅
If the package was bumped by lerna – we should release it. I prefer having "No updates" for the packages that we release.

Reply:
> If you can apply the same logic as what we chose for the charts on today's release it would be great

No manually updating the changelog for version 7.12.1

If the script can be improved I'm all in favor of doing it, but later on :laughing:

Reply:
I agree that we should signal the lack of specific changes in such cases. 🤔 
Can we add `No changes.` as we once did?
This would allow us to avoid the need to track the last release and put that in. 🙈